buildout.cfg

```diff
@@ -18,6 +18,9 @@ parts +=
     rebuild_i18n-sh
     robot
 
+test-eggs +=
+    plone4.csrffixes
+
```

Comment:
@rodfersou no, we have to test this in all Plone versions supported by the package; figuring out how is the hard part.